SEARCHSPEC.md

```diff
@@ -918,11 +918,11 @@ The difference between the two formats is the way in which the phenopacket json
 
 # Appendix A: SQL Grammar
 
-ANTLR grammar for GA4GH Search (based on PrestoSQL v. 323, ASL 2.0 license), with the DML and DDL parts removed:
+ANTLR grammar for Data Connect (based on PrestoSQL v. 323, ASL 2.0 license), with the DML and DDL parts removed:
 
 
 ```
-grammar Ga4ghSearch;
+grammar Ga4ghData Connect;
```

Comment:
Should be `grammar DataConnect` (I believe the space is a syntax error)

Reply:
Good catch! I intended "Search", as in the search endpoint of Data Connect, but let's go with Data Connect.